### PR TITLE
feat: add events and effect to add claim condition annotations

### PR DESCRIPTION
**Pull request by nmpsilva**
Adds a new event and a new effect to support annotations on Claim Modal, for conditions.

### CODE REVIEW

**Comment by nmpsilva** [comment]
@aduane I have no idea if we're following any kind of numbering for the events/effects, so lmk if I should change it

File: protobufs/canvas_generated/messages/events.proto [hunk 1]
```diff
@@ -771,6 +771,7 @@ enum EventType {
   // VITALS_COMMAND__POST_EXECUTE_ACTION = 52011;
 
   PATIENT_CHART__CONDITIONS = 100000;
+  CLAIM__CONDITIONS = 101000;
```
Comment (aduane) [inline protobufs/canvas_generated/messages/events.proto:774]: I think these should be named more specifically. They fire when these are being processed in a graphql resolver, right?

Comment (nmpsilva) [reply protobufs/canvas_generated/messages/events.proto:774]: Not this one; This one fires right after the serializer serializes data. :\

**Comment by nmpsilva** [comment]
@aduane pinging you for a review.

**Review by aduane — APPROVED** [review]

**Pull request merged**